ui/components/component-library/tag-url/README.mdx

```diff
@@ -120,9 +130,9 @@ import { TagUrl } from '../../ui/component-library/tag-url';
   label="app.uniswap.org"
 />
 <TagUrl
-  src="https://metamask.io/icons/icon-48x48.png?v=48400a28770e10dd52a8c0e539aeb282"
+  src="https://metamask.github.io/test-dapp/metamask-fox.svg"
   showLockIcon
-  label="metamask.io"
+  label="metamask.github.io"
 />
 <TagUrl
   src=""
```

Comment:
Made some small updates to the docs page to show missing images and improve the sentence structure of some sections